### PR TITLE
Auqa 85 UI popup verification

### PR DESCRIPTION
**Pull request by abhijitvigs**

### CODE REVIEW

**Review by abhijitvigs — COMMENTED** [review]
All done

File: PageObjects/vxAIEngine.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# VX AIEngine - Site Editor Web Elements
```
Comment (ananiac) [inline PageObjects/vxAIEngine.py:1]: Rename vxAIEngine.py  to Ex: systemconsole

File: Resources/UIResources/uiPopupResources.robot [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    ${popup_message}=  get text  ${suppress_alarm_popup_message}
+    should be equal	 ${popup_message}  ${test_input}[suppress_alarm_popup_message]
+    log to console  Popup message is "${popup_message}"
+    click element  ${suppress_alarm_ok_button}
+    selectAlarm
+    sleep  ${load_time}
+    close browser
+
+selectAlarm
+    click element  xpath=//table[3]/tbody/tr/td[3]/div
```
Comment (ananiac) [inline Resources/UIResources/uiPopupResources.robot:45]: Move this to the pageobject file

**Review by ananiac — COMMENTED** [review]
1) Update wednesdaysuite and thursdaysuite with the popup message testcase
2) Add the test execution command to testexecutioncommands

File: PageObjects/systemconsole.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# VX AIEngine - Site Editor Web Elements
```
Comment (ananiac) [inline PageObjects/systemconsole.py:1]: Follow the naming convention- systemconsole as systemConsole

**Review by ananiac — COMMENTED** [review]
in testexecutioncommands combine Report is missing the poupxml
popup excution caommand is incorrect

**Review by ananiac — COMMENTED** [review]
1) group name is missing in the command in testexecutioncommands sheet
robot --name 'Popup'  --reporttitle "Popups"  --outputdir Reports --output uiPopup.xml --variable environment:config37 /home/fc/automation/AuQA/Testcases/UITests/uiPopupTest.robot
it should be 
robot --name 'Popup'  --reporttitle "Popups"  --outputdir Reports --output uiPopup.xml --variable environment:config37 -v groupname:General-test /home/fc/automation/AuQA/Testcases/UITests/uiPopupTest.robot

2) There is no yes/no drop down in all the three suite sheet for B8 column

**Pull request merged**